src/docs/asciidoc/examples.adoc

```diff
@@ -0,0 +1,165 @@
+ Device Summary Response
+------------------------
+
+Based on the the device type identified , the summary response schema could be any one of the following..
```

Comment:
remove repeated "the" and space before coma.  Add a colon instead of ..

Reply:
done